src/material-experimental/mdc-dialog/dialog.scss

```diff
@@ -1,8 +1,34 @@
 @use '@material/dialog' as mdc-dialog;
+@use '@material/dialog/dialog-theme' as mdc-dialog-theme;
 @use '../mdc-helpers/mdc-helpers';
 @use './mdc-dialog-structure-overrides';
 @use '../../cdk/a11y';
 
+// Theme map with values for variables that will be overriden in the theme.
+// MDC's theming system requires non-null values for the slots to be inserted
+// and included as default values.
+$dialog-initial-theme: (
```

Comment:
Do we actually need this? For the other components MDC provided a `$light-theme` with some default values that I used outside for the `theme-styles` mixin.

Reply:
Also if we do end up keeping this variable, the name should be with an underscore so that it isn't exported by Sass.

Reply:
Unfortunately yes - their variable contains the keys but with null values, which means their theme mixin skips inserting the slots. They plan to eventually provide a variable with filled values, but not until at least a couple quarters from now.

Changed variable name to underscore-prefix - thanks for catching that

Reply:
Here's their variable definition - https://github.com/material-components/material-components-web/blob/master/packages/mdc-dialog/_dialog-theme.scss#L36

They say that null (or any other falsy) values don't insert slots for backwards compatibility reasons. I wish that wasn't the case

Reply:
Hmm, that is unfortunate and a little fragile since we won't have a way of knowing when they introduce a new variable.